calico_versioned_docs/version-3.27/reference/installation/tigerastatus.mdx

```diff
@@ -0,0 +1,40 @@
+---
+description: Descriptions of Tigera Status fields
+---
+
+# Tigera status
+
+Installing {{prodname}} on your Kubernetes cluster is managed by the Tigera operator. The Tigera operator is deployed as a ReplicaSet in the `tigera-operator` namespace, and records status in a custom resource named, `tigerastatus`. The operator get its configuration from several Custom Resources (CRs); the central one is the Installation CR.
+
+To check tigerastatus, use the following command.
+
+`kubectl get tigerastatus`
+
+## Log storage controllers
+
+The following controllers comprise the log storage component. Note the following:
+
+- All controllers are required and are not configurable
+- Controllers work together to implement log storage functionality, but are reconciled independently
+- Controllers do not communicate directly with each other; they communicate through the Kubernetes API
+
+To check log collector status, run the following command:
+
+`kubectl get logcollector.operator.tigera.io tigera-secure`
+
+The following table lists controller descriptions.
+
+| Tigera status field         | Description                                                  |
+| --------------------------- | ------------------------------------------------------------ |
+| log-storage                 | Provides initialization validation and defaulting for log storage, creates base namespaces for controllers to deploy to, and provides status and condition updates on other controllers. |
```

Comment:
Sorry, I should have been more clear in my original message.

All of the `log-storage-x` Tigera status instances are for enterprise, Calico OSS doesn't have any of these.

The ones that OSS has are:

- `calico`
- `apiserver`

And that's it IIRC

Reply:
I've made the changes only to CE (3.18 and 3.19) for now. Just added the line about log-storage-elastic about running in standalone and management clusters.